runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -346,7 +346,8 @@ NNFW_STATUS nnfw_session::prepare()
     return NNFW_STATUS_INVALID_STATE;
   }
 
-  if (!_subgraphs || !primary_subgraph() || primary_subgraph()->isBuildingPhase())
+  auto primary_subgraph = _subgraphs->primary();
+  if (!_subgraphs || !primary_subgraph || primary_subgraph->isBuildingPhase())
```

Comment:
It checks `_subgraphs` is `nullptr` after `_subgraphs->primary()`

Reply:
You're right and I found this redundant as it is all checked from the right above. So I removed the whole checks here.